Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN \
   apk --update --upgrade \
     add ca-certificates --virtual .buildDeps git ca-certificates openssl g++ make python3 linux-headers \
   && update-ca-certificates \
+  && npm i -g corepack@latest \
```

Comment:
this is a hack to solve whatever issue there is. please ensure that configuration is correct for corepack/pnpm/semaphore, lmk if you need help identifying the problem